tests/test_to_string.cc

```diff
@@ -0,0 +1,30 @@
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
```

Comment:
It's been customary for all the test files to also have the preamble as the main code. I don't see an explicit recommendation on the dev guide about this though.